site.go

```diff
@@ -103,6 +112,35 @@ func LoadSiteFromFile(path string) (*Site, error) {
 		return nil, err
 	}
 
+	// set up private key for thumbor+cloudfront, if applicable
+	if s.ResizingService == "thumbor+cloudfront" && s.AWS_CLOUDFRONT_PRIVATE_KEY_PATH != "" && s.AWS_CLOUDFRONT_PRIVATE_KEY_PAIR_ID != "" {
```

Comment:
Can the server work with resizing service `thumbor+cloudfront` with the keys set to empty strings?

If not, I feel we should move this check into `Site.IsValid()` and here continue with the assumption that all required config values are present.

Reply:
fair, i'm already checking it there but we can at this point assume it exists